docs/configuration_and_processing.md

```diff
@@ -48,23 +47,23 @@ use Ssch\TYPO3Rector\Configuration\Typo3Option;
 use Ssch\TYPO3Rector\Rector\General\ConvertImplicitVariablesToExplicitGlobalsRector;
 use Ssch\TYPO3Rector\Rector\General\ExtEmConfRector;
 use Ssch\TYPO3Rector\Set\Typo3LevelSetList;
+use Ssch\TYPO3Rector\Set\Typo3SetList;
 
 return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->sets([
-        Typo3LevelSetList::UP_TO_TYPO3_11
-    ]);
-
-    // In order to have a better analysis from phpstan we teach it here some more things
-    $rectorConfig->phpstanConfig(Typo3Option::PHPSTAN_FOR_RECTOR_PATH);
 
-    // FQN classes are not imported by default. If you don't do it manually after every Rector run, enable it by:
-    $rectorConfig->importNames();
+    // If you want to override the number of spaces for your typoscript files you can define it here, the default value is 4
+    // $parameters = $rectorConfig->parameters();
+    // $parameters->set(Typo3Option::TYPOSCRIPT_INDENT_SIZE, 2);
 
-    // Disable parallel otherwise non php file processing is not working i.e. typoscript
-    $rectorConfig->disableParallel();
```

Comment:
These are important lines that are lost now. They need to be added again.

Reply:
done,